README.md

```diff
@@ -1,167 +1,19 @@
-# Scope: what and why
+# Variable Scope: Lab
 
-Imagine if all parts of your code knew about all the variables in all other parts of your code. You'd have to take great care not to accidentally reuse a variable name in multiple places -- `point` may be an x-y coordinate in one place but lat-long elsewhere, and you'd have to keep that all in mind. Yikes!
+## Objectives
 
-Luckily, the visibility of classes, methods, and variables is controlled by what is called **scope**. The things that a particular piece of code can see are called "in scope". Those it cannot are "out of scope".
+1. Use method arguments to pass values horizontally between contexts.
+2. Write loops and conditionals with an awareness of scope.
+3. Pull a value out of a localized context by declaring it prior to the localization.
 
-The ins and outs of scope vary dramatically from language to language. Luckily Objective-C keeps it pretty simple.
+## Intructions
 
-# The rules
+In this lab we're going to get some practice scope by working with different local contexts. In addition to passing values horizontally through method arguments, you'll have to navigate writing some loops and conditionals that localize your context further.
```

Comment:
woah this paragraph has *so much* jargon. I think the only jargon we need to make sure they understand is the term "scope". "In this lab we're going to practice with variable scope. You'll be passing variables around and ensuring the right variables are in scope in the right places in and out of your loops and conditionals"